app/ante.go

```diff
@@ -79,7 +76,7 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 		ante.NewTxTimeoutHeightDecorator(),
 		ante.NewValidateMemoDecorator(options.AccountKeeper),
 		ante.NewConsumeGasForTxSizeDecorator(options.AccountKeeper),
-		ante.NewDeductFeeDecorator(options.AccountKeeper, options.BankKeeper, options.FeegrantKeeper),
```

Comment:
Passing in an addtional arg that's just the TxFeeChecker which is already an option

Reply:
cc @Kbhat1 

from this PR: https://github.com/sei-protocol/sei-cosmos/pull/23/files